rb/spec/integration/selenium/webdriver/chrome/options_spec.rb

```diff
@@ -24,34 +24,6 @@ module WebDriver
     module Chrome
       describe Options, only: {browser: :chrome} do
         subject(:options) { Options.new }
-
-        it 'passes emulated device correctly' do
```

Comment:
Is there any reason why these specs should be removed?

Reply:
because they apply to both chrome & edge_chrome, so they run in the chromium directory:
https://github.com/SeleniumHQ/selenium/pull/8497/files#diff-33481e2e9dad5e27918bd2d2660a9f9fL28